workshops/konami_code/README.md

```diff
@@ -0,0 +1,478 @@
+
+### 3) JavaScript
+
+Go to your `script.js` file. First let's link the HTML elements to JavaScript. Type the following code:
+
+```js
+const text = document.querySelector(".text");
+const secretDiv = document.querySelector('.secret-div');
+```
+
+**Explanation:** We linked the `<h1>` and also the `secret-div` tags to the two variables `text` and `secretDiv` respectively.
+
+**What is `querySelector()`?** The `querySelector()` method returns the first element that matches a specified CSS class(s) in the document.
+
+Learn more about [query selector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector).
+
+Now we will declare two arrays. The first array will contain the keys pressed by the user and the second array will contain the keys which will be the Konami Code.
+
+```js
+const keys = [];
+const secretCode = ["h", "a", "c", "k"];
+```
+Here, the `secretCode` array contains the keys which will be the Konami Code. For this workshop's sake, our code will be "hack". You can change it to anything you like. Here are some examples:
+
+```js
+["KeyUp", "KeyDown", "KeyLeft", "KeyRight"]
+["KeyUp", "a", "KeyLeft", "z", "KeyDown"]
+["Shift", "Control", "Enter", "b"]
+["a", "a", "b", "b", "c", "d", "e", "KeyLeft"]
+```
+
+Next, we will make use of the confetti.js library which we have imported in our project. We will first declare some settings for the confetti. After we declare them, we will create a new `ConfettiGenerator` object which will help us implement the confetti on the site.
+
+```js
+const confettiSettings = { target: 'my-canvas', size: 1.4, max: 150 };
+const confetti = new ConfettiGenerator(confettiSettings);
+```
+**Explanation:** The settings for the confetti is an object. It is made with reference to [this](https://github.com/Agezao/confetti-js/) site. There are many more options which you can pass to the settings as you wish.
+
+The `target` attribute is used to specify the canvas with the id `my-canvas` on which the confetti will appear.
+
+The `size` attribute specifies the size for the confetti particles.
+
+The `max` attribute specifies the The number of props(confetti) to be rendered.
+
+The `ConfettiGenerator` contains a `render()` method and when it is called, we will be able to see confetti on our page.
+
+For more information on these attributes, [check here](https://github.com/Agezao/confetti-js/).
+
+Your code so far:
+```js
+const text = document.querySelector(".text");
+const secretDiv = document.querySelector('.secret-div');
+
+const keys = [];
+const secretCode = ["h", "a", "c", "k"];
+
+const confettiSettings = { target: 'my-canvas', size: 1.4, max: 150 };
+const confetti = new ConfettiGenerator(confettiSettings);
+```
+
+Next we will implement a function to check which keys are pressed by the user and when the keys match our Konami Code, it will get triggered.
+
+The function will take an 'event' attribute which will give us the key pressed by the user.
+
```

Comment:
```suggestion
Next, we will implement a function to check which keys are pressed by the user, and when the keys match our Konami Code, it will get triggered.

The function will take an 'event' attribute which will give us the key pressed by the user.

```